code/modules/antagonists/nukeop/equipment/borgchameleon.dm

```diff
@@ -62,6 +62,8 @@
 			to_chat(user, "<span class='notice'>\the [src] is recharging.</span>")
```

Comment:
you might want to remove the "\t" here and replace it with a "T"

it's not your code here, but still.

Reply:
I would rather not mess with default TG code unless strictly necessary.